common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
@alisan617 FYI, you should follow this same pattern being discussed above in your PR.

But @robrap, I'm still confused why there is a preference of ${msg | n, decode.utf8} over HTML(). Can you explain more?